pyproject.toml

```diff
@@ -41,7 +41,7 @@ classifiers = [
 
 
 [tool.poetry.dependencies]
-python = ">=3.10, <3.11"
+python = "^3.10"
```

Comment:
Could be missing something but does this also include 3.12?

Reply:
For some reason I thought 3.12 wasn't live yet, but [it released 6 months ago](https://devguide.python.org/versions/), so I've updated tox to test 3.12 as well.

In general my assumption was that Python wasn't going to break backward compatibility for future 3.XX releases, so it was safe to assume Model Gauge will work going forward.